auth/auth-router.js

```diff
@@ -1,11 +1,52 @@
 const router = require('express').Router();
+const bcrypt = require('bcryptjs')
+const jwt = require('jsonwebtoken')
 
-router.post('/register', (req, res) => {
+const Jokes = require('../jokes/jokes-model')
+
+const generateToken = user => {
+  const payload = {
+    sub: user.id,
+    username: user,username,
+  }
+  const options = {
+    expiresIn: '5m',
+  }
```

Comment:
I really like how you made the variables outside and then used it all together to create the token.  Much cleaner way of doing it